replay-assets/replay_command_handlers.js

```diff
@@ -1725,12 +1725,91 @@ function getInternalFunctionLocationProp(cdpProperties) {
   return getInternalProp(cdpProperties, '[[FunctionLocation]]');
 }
 
+/**
+ * String utility for function parameter parsing.
+ */
+function hasSubstringAt(haystack, needle, i) {
+  return haystack.substring(1 + i - needle.length, 1 + i) === needle;
+}
+
+/**
+ * [RUN-3146] Extract parameter names from the description.
+ * @param {string} s 
+ * @return {string[]}
+ */
+function extractFunctionParameterNames(s) {
```

Comment:
At a minimum we should have some unit tests. 

It might be helpful to include a link to where chrome devtools implements this as well.

Reply:
Yep. Read the TODO here - I found that after I created this PR:
https://linear.app/replay/issue/RUN-3146/function-object-preview-is-missing-functionparameternames-in-chromium#comment-fab95193
![image](https://github.com/replayio/chromium/assets/282899/63fd02a6-2dfc-481a-a795-6ffc2becbf4a)

Reply:
> It might be helpful to include a link to where chrome devtools implements this as well.


Its all in the link. Also: You asked me this very question before, and I already replied that its already in the issue:
![image](https://github.com/replayio/chromium/assets/282899/305ed716-0f37-483e-ab19-631357f19af3)